<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformDialog</class>
 <widget class="QDialog" name="TransformDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transform</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Translate X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Translate Y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Translate Z</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Rotate X</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Rotate y</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Rotate Z</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Scale</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="mp_spin_translate_x">
       <property name="minimum">
        <double>-200000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="mp_spin_scale">
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="mp_spin_translate_y">
       <property name="minimum">
        <double>-200000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>200000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="mp_spin_translate_z">
       <property name="minimum">
        <double>-200000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>200000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="mp_spin_rotate_x">
       <property name="minimum">
        <double>-180.000000000000000</double>
       </property>
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="mp_spin_rotate_y">
       <property name="minimum">
        <double>-180.000000000000000</double>
       </property>
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="mp_spin_rotate_z">
       <property name="minimum">
        <double>-180.000000000000000</double>
       </property>
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="mp_button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mp_button_box</sender>
   <signal>accepted()</signal>
   <receiver>TransformDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp_button_box</sender>
   <signal>rejected()</signal>
   <receiver>TransformDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
